conversion_rules_export/csv/aws-cur.csv

```diff
@@ -66,16 +66,16 @@ CommitmentDiscountType,1,NA,Not Defined,SQL_CONDITION,"conditions:
 - WHEN ""reservation/ReservationARN"" is not null THEN 'Reserved Instances (RI)'
 default_value: 'null'
 "
-EffectiveCost,1,NA,Not Defined,SQL_QUERY,"SELECT *, CASE
-    WHEN (""lineItem/LineItemType"" = 'SavingsPlanCoveredUsage') THEN ""savingsPlan/SavingsPlanEffectiveCost""
-    WHEN (""lineItem/LineItemType"" = 'SavingsPlanRecurringFee') THEN (""savingsPlan/TotalCommitmentToDate"" - ""savingsPlan/UsedCommitment"")
-    WHEN (""lineItem/LineItemType"" = 'SavingsPlanNegation') THEN 0
-    WHEN (""lineItem/LineItemType"" = 'SavingsPlanUpfrontFee') THEN 0
-    WHEN (""lineItem/LineItemType"" = 'DiscountedUsage') THEN ""reservation/EffectiveCost""
-    WHEN (""lineItem/LineItemType"" = 'RIFee') THEN (""reservation/UnusedAmortizedUpfrontFeeForBillingPeriod"" + ""reservation/UnusedRecurringFee"")
-    WHEN ((""lineItem/LineItemType"" = 'Fee') AND (""reservation/ReservationARN"" <> '')) THEN 0
-    ELSE ""lineItem/UnblendedCost""
-END AS EffectiveCost FROM {{ TABLE_NAME }}
+EffectiveCost,1,NA,Not Defined,SQL_QUERY,"""SELECT *, CASE\n    WHEN (\""lineItem/LineItemType\"" = 'SavingsPlanCoveredUsage')\
```

Comment:
Are escaping new lines and `"` useful here somehow? Previous results are cleaner.

Reply:
Oh yes thats a good point, let me see if I introduced this by mistake.

Reply:
I think that I don't get it. Do you mean a generic plan related to this PR? Or it is something else?